docs/staff/coaches/exercises.md

```diff
@@ -0,0 +1,101 @@
+
+From here you can override various the normal flow of Jarvis for a student.
+Using the menu in the right bottom hand corner you can enroll a student in any module you'd like.
+
+Clicking on a module the student's already enrolled in you can view that students progress within that module.
+
+![student progress module example](/img/staff/coaches/exercises/student-progress-module.png)
+
+Depending on the state an exercise there will be a menu on the right hand side of the exercise to allow you to:
+
+* View the exercise like that student (to see their BLAST results for example)
+* Unlock a locked exercise
+* [Review a self diagnostic with a student](#self-diagnostic)
+* Make a student redo an exercise
+
+### Team exercises 👥
+
+If the exercise happens to be a exercise where students work together.
+You will additionally see the controls to manually control teams.
+
+In the case of a team exercise that requires 4 students,
+Jarvis will by default start a team with the first student that arrives.
+The second student that arrives will be added to this team,
+and the exercise will start once the required number has been reached.
+
+You can manually override this by, disbanding a team and then manually enrolling any team you'd like.
+This can also be used to deal with uneven group sizes,
+say you have 21 students in a group you don't want the last 3 to get stuck,
+so you can manually start them as a group of 3.
+This is done by disbanding the team and them manually enrolling.
+
+## Self diagnostic
+
+Self diagnostics are a special kind of exercise,
+in which students take a little quiz individually to verify their comprehension of a specific topic.
+
+This is often used for more theoretical subjects and/or subjects that are not properly verified by practical exercises.
+
+If the student fails this self diagnostic, depending on location settings,
+they might be forced so seek out a coach to help them understand where they went wrong.
+
+![student progress self diagnostic example](/img/staff/coaches/exercises/student-progress-self-diagnostic.png)
+
+From the module overview you can review a self diagnostic with a student.
+You'll be able to see the correct answer's (and hide them when going through the quiz with the student).
+
+![self diagnostic review example](/img/staff/coaches/exercises/self-diagnostic.png)
+
+After going through the quiz with the student you can reopen the self diagnostic from the module overview.
+After which the student can retake the quiz.
+(please note the student can get different questions on the same subjects)
+
+## Settings
+
+Most of the settings of an exercise/module are part of that content. 
+The module contains things like:
+* Can a student download a certificate?
+* How many open reviews can a student have? (default is 7)
+
+The exercise settings contain things like:
+* What files should a student hand in?
+* How should a student hand in those files? (Git or file upload)
```

Comment:
Maybe dat we hier willen linken naar een stukje docs waarin we voor niet IT scholen uitleggen wat Git is.

En daar dan ook voor IT scholen wat pointers geven mbt veel voorkomende problemen van studenten icm Git

Reply:
Ja lijkt me goed, ik maak hier ff story voor aan.